packages/libs/pactjs-generator/src/contract/parsing/utils/parser-utilities/rule.ts

```diff
@@ -0,0 +1,33 @@
+import { ExWrappedData } from '../dataWrapper';
+import { IPointer } from '../getPointer';
+
+// eslint-disable-next-line @rushstack/typedef-var
+export const FAILED = Symbol('FAILED');
```

Comment:
```suggestion
export const FAILED: symbol = Symbol('FAILED');
```
?

Reply:
I want a unique symbol. but `:symbol` just mention its a symbol not a specific symbol and then it breaks all of the places that work with FAILED symbol.
So basically 
```typescript
const A = Symbol('A');
const B = Symbol('B');
// This comparison appears to be unintentional because the types 'typeof A' and 'typeof B' have no overlap.
if (A === B) {
}
```
but this one does not throw the ts error
```typescript
const A:symbol = Symbol('A');
const B:symbol = Symbol('B');
// type script cant understand these symbols are dufferent
if (A === B) {
}
```

Reply:
And what is the point of adding a type for primitive types at all. I believe its redundant and sometimes just does not let typescript to refer types completely .
```typescript
// typescript narrows the type to "HELLO" ; much better type for a const
const A = "HELLO"

// this is just a string
const B:string = "HELLO"